src/Alpha_complex/include/gudhi/Alpha_complex_3d.h

```diff
@@ -225,30 +225,35 @@ class Alpha_complex_3d {
    * Must be compatible with double. */
   using FT = typename Alpha_shape_3::FT;
 
-  /** \brief Gives public access to the Point_3 type. Here is a Point_3 constructor example:
+  /** \brief Gives public access to the Bare_point_3 (bare aka. unweighed) type.
+   * Here is a Bare_point_3 constructor example:
 \code{.cpp}
 using Alpha_complex_3d = Gudhi::alpha_complex::Alpha_complex_3d<Gudhi::alpha_complex::complexity::SAFE, false, false>;
 
 // x0 = 1., y0 = -1.1, z0 = -1..
-Alpha_complex_3d::Point_3 p0(1., -1.1, -1.);
+Alpha_complex_3d::Bare_point_3 p0(1., -1.1, -1.);
 \endcode
    * */
-  using Point_3 = typename Kernel::Point_3;
+  using Bare_point_3 = typename Kernel::Point_3;
 
   /** \brief Gives public access to the Weighted_point_3 type. A Weighted point can be constructed as follows:
 \code{.cpp}
-using Weighted_alpha_complex_3d =
-  Gudhi::alpha_complex::Alpha_complex_3d<Gudhi::alpha_complex::complexity::SAFE, true, false>;
+using Weighted_alpha_complex_3d = Gudhi::alpha_complex::Alpha_complex_3d<Gudhi::alpha_complex::complexity::SAFE, true, false>;
 
 // x0 = 1., y0 = -1.1, z0 = -1., weight = 4.
-Weighted_alpha_complex_3d::Weighted_point_3 wp0(Weighted_alpha_complex_3d::Point_3(1., -1.1, -1.), 4.);
+Weighted_alpha_complex_3d::Weighted_point_3 wp0(Weighted_alpha_complex_3d::Bare_point_3(1., -1.1, -1.), 4.);
 \endcode
  *
  * Note: This type is defined to void if Alpha complex is not weighted.
  *
  * */
   using Weighted_point_3 = typename Triangulation_3<Kernel, Tds, Weighted, Periodic>::Weighted_point_3;
 
+  /** \brief `Alpha_complex_3d::Point_3` type is either a `Alpha_complex_3d::Bare_point_3` (Weighted = false) or a
+   * `Alpha_complex_3d::Weighted_point_3` (Weighted = true).
+   */
+  using Point_3 = typename std::conditional<Weighted, Weighted_point_3, Bare_point_3>::type;
```

Comment:
Is it just `typename Alpha_shape_3::Point`?

Reply:
From my point of view, it is better to explain that Point_3 is a Bare_point_3 or a Weighted_point_3, than using a nested Alpha_shape type, no ? I might be wrong...

Reply:
We could do both: for instance keep your doc, and change the code.

Reply:
Done on 6d2e5d1cbdecd86ad4d56fc8902542caca2cf2bd